lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -541,6 +541,7 @@ def add_external_three_d_secure_auth_data(post, options = {})
       def setup_future_usage(post, options = {})
         post[:setup_future_usage] = options[:setup_future_usage] if %w(on_session off_session).include?(options[:setup_future_usage])
         post[:off_session] = options[:off_session] if off_session_request?(options)
+        post[:expand] = ['charges.data.balance_transaction'] if post[:setup_future_usage] || post[:off_session]
```

Comment:
It also works for regular transactions, just tested without `if post[:setup_future_usage] || post[:off_session]`.

Reply:
Idk why it errored out when you tried it last time.

Reply:
Also for unit test maybe you can just check that `post[:expand] = ['charges.data.balance_transaction']` gets added to params

Reply:
Thanks for the review and suggestions @almalee24 ! I've re-requested you for review after making the changes as well. Unsure myself why I was getting errors last time